CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,60 @@
+
+We like these hashtags: #noCodeSnobs (because we value community over efficiency), #newKidLove (because we all started somewhere), #unassumeCore (because we don't assume knowledge), and #BlackLivesMatter (because of course).
+
+
+**In practice:**
+
+
+-  We are not code snobs. We do not assume knowledge or imply there are things that somebody should know.
+-  We insist on actively engaging with requests for feedback regardless of their complexity.
+-  We welcome newcomers and prioritize the education of others. We strive to approach all tasks with the enthusiasm of a newcomer. Because we believe that newcomers are just as valuable in this effort as experts.
+-  We consistently make the effort to actively recognize and validate multiple types of contributions.
+-  We are always willing to offer help or guidance.
+
+
+**In times of conflict:**
+
+-  We listen.
+-  We clearly communicate while acknowledging other's feelings.
+-  We admit when we're wrong, apologize, and accept responsibility for our actions.
+-  We are continuously seeking to improve ourselves and our community.
+-  We keep our community respectful and open.
+-  We make everyone feel heard.
+-  We are mindful and kind in our interactions.
+
+
+**In the future:**
+
+-  The future is now.
+
+
+### Contribute
+
+Our community is always looking for enthusiasts to help in all different ways. 
+
+-  **Develop**. [GitHub](https://github.com/ml5js/ml5-library) is the main place where code is collected, issues are documented, and discussions about code are had. Check out the [contributing documentation to get started](https://github.com/ml5js/ml5-library/blob/development/CONTRIBUTING.md) with developing the library.
+-  **Document**. Everyone loves documentation. Help is needed porting examples, and adding documentation, and creating tutorials.
+-  **Teach**. Teach a workshop, a class, a friend, a collaborator! Tag [@ml5js on Twitter](https://twitter.com/ml5js?lang=en) and we will do our best to share what you're doing.
+-  **Create**. ml5.js is looking for designers, artists, coders, programmers to bring your creative and amazing work to show and inspire other people. Submit your work to info@ml5js.org.
+-  **Donate**. ml5.js is a free and open source and made by artists. While ml5.js is not currently accepting donations, you can help support the development of ml5.js through a donation to other adjacent open source projects and endeavors such as the [Processing Foundation](https://processingfoundation.org/support) and [The Coding Train](https://www.patreon.com/codingtrain).
```

Comment:
I feel a little strange encouraging people to donate to Coding Train from here, I think Processing Foundation makes more sense. We could also point to the Red Burns Scholarship fund if that seems appropriate? https://itp.nyu.edu/redburnsfund/
```suggestion
-  **Donate**. ml5.js is a free and open source and made by artists. While ml5.js is not currently accepting donations, you can help support the development of ml5.js through a donation to other adjacent open source projects and endeavors such as the [Processing Foundation](https://processingfoundation.org/support).
```

Reply:
@shiffman - oh wow! yes, I think it makes sense to have a link to the Red Burns Scholarship esp. since ITP is the birthplace of ml5  and the students are very much major contributors when it comes to testing, developing, etc. 

Do you think it makes sense to maybe reference other orgs here like:
* Data and Society - https://datasociety.net/engage/#donate
* STOP (surveillance technology oversight project) - https://www.stopspying.org/ 
* Code for Science and Society - https://codeforscience.org/
* or other tech for good orgs?